pyproject.toml

```diff
@@ -152,6 +152,7 @@ pytest-xdist = "*"
 pytest-timeout = "*"
 pytest-timeouts = "*"
 pytest-random-order = "*"
```

Comment:
@jnewb1 can we remove `pytest-random-order`? It's slightly less updated and not used unless `--random-order` is specified, which it's not.

Reply:
we'd like to add it in the future, it just causes some issues with some tests right now. does it cause any issues?

Reply:
Why did we add without enabling? https://github.com/commaai/openpilot/pull/30166

Reply:
it was so I could test the random order plugin in CI without having to rebuild the whole image every time

Reply:
we do want to add it eventually so ideally we just keep it installed until we are ready to use it (when all the tests are fully independent and can be run in any order)

Reply:
No issues with either with `-m 'not slow'`

Reply:
no issues enabling it you mean? the issues that happened were somewhat rare, like 1/10 runs in CI.  https://github.com/commaai/openpilot/pull/30134

Reply:
well we should fix that, if that's the case. no issues enabling both for me on PC tests. and it seems like both truly randomize the tests (my two bootlog failures occur in different places), so I'll remove `pytest-random-order` since `pytest-randomly` can do the same thing.